apps/ledger-live-desktop/src/newArch/features/DynamicContent/components/PortfolioContentCards/Slide.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { memo } from "react";
+import { useHistory } from "react-router-dom";
+
+import { PortfolioContentCard as Card } from "@ledgerhq/react-ui";
+import { openURL } from "~/renderer/linking";
+import type { PortfolioContentCard } from "~/types/dynamicContent";
+import type { CarouselActions } from "../../types";
+import LogContentCardWrapper from "../LogContentCardWrapper";
+
+export default memo(Slide);
+
+type Props = PortfolioContentCard & CarouselActions & { index: number };
+
+function Slide({ logSlideClick, dismissCard, index, ...card }: Props) {
+  const history = useHistory();
+
+  const handleClose = () => dismissCard(index);
+  const handleClick = () => {
+    logSlideClick(card.id);
+
+    if (card.path) {
+      history.push({ pathname: card.path, state: { source: "banner" } });
+    } else if (card.url) {
+      openURL(card.url);
+    }
+  };
+
+  return (
```

Comment:
while it's a new component in the `newArch` can you add a integ test for it please ?

Reply:
This component is being tested as part of this integration test: [LLD/features/DynamicContent/__integrations__/PortfolioContentCards.test.tsx](https://github.com/LedgerHQ/ledger-live/pull/8397/files/5fc7206675b02e9cab60cc5863fb013a52ffa91f#diff-18d150d3c03402746d982dd06cd632fade99116a9a97b8f231fdc0eb90929b7c). Actually it's almost all of what this test covers because the carousel relies on styling and therefore is not really tested and as mentioned in the test comments `LogContentCardWrapper` relies on intersection observers making it untestable too.